crates/yttrium/src/sign/mod.rs

```diff
@@ -124,47 +131,40 @@ pub enum RespondError {
     Internal(String),
 }
 
-// #[cfg(target_arch = "wasm32")]
-// struct WebWebSocketWrapper {
-//     // rx: tokio::sync::mpsc::UnboundedReceiver<String>,
-//     // tx: tokio::sync::mpsc::UnboundedSender<String>,
-// }
-
-struct WebSocketState {
-    // #[cfg(not(target_arch = "wasm32"))]
-    // stream: WebSocketStream<MaybeTlsStream<TcpStream>>,
-    // #[cfg(target_arch = "wasm32")]
-    // stream: WebWebSocketWrapper,
-    // message_id: u64,
-    request_tx: tokio::sync::mpsc::UnboundedSender<(
-        Params,
-        tokio::sync::oneshot::Sender<Result<Response, RequestError>>,
-    )>,
-}
-
+#[cfg(feature = "uniffi")]
 #[uniffi::export(with_foreign)]
 pub trait SignListener: Send + Sync {
     //TODO: add on_session_ping, on_session_update, on_session_event, on_session_extend, on_session_disconnect etc.
-    fn on_session_request(&self, topic: String, session_request: SessionRequestJsonRpcFfi);
+    fn on_session_request(
+        &self,
+        topic: String,
+        session_request: SessionRequestJsonRpcFfi,
+    );
 }
 
+#[cfg(feature = "uniffi")]
 #[uniffi::export(with_foreign)]
-pub trait SessionStore: Send + Sync {
+pub trait SessionStoreFfi: Send + Sync {
     fn add_session(&self, session: SessionFfi);
     fn delete_session(&self, topic: String);
     fn get_session(&self, topic: String) -> Option<SessionFfi>;
     fn get_all_sessions(&self) -> Vec<SessionFfi>;
 }
 
+pub trait SessionStore: Send + Sync {
+    fn add_session(&self, session: Session);
+    fn delete_session(&self, topic: String);
+    fn get_session(&self, topic: String) -> Option<Session>;
+    fn get_all_sessions(&self) -> Vec<Session>;
+}
+
```

Comment:
Don't fully understand why do we need SessionStore and SessionStoreFfi traits? Why can't we just map the types instead of having two traits?